shell/utils/auth.js

```diff
@@ -161,3 +166,144 @@ export const canViewResource = (store, resource) => {
 
   return !!validResource;
 };
+
+// ************************************************************
+//
+// BELOW ARE METHODS THAT ARE A PART OF THE AUTHENTICATED MIDDLEWARE REMOVAL. THIS IS A TEMPORARY HOME FOR THESE UTILS AND SHOULD BE REWRITTEN, MOVED OR DELETED.

```

Comment:
Thanks for mentioning that this is a first clean round.
If you are not planning to work on it, would it be ok to add an issue with a TODO, so we can track it down?
Otherwise is enough a PR as usual.

Reply:
Updated the code and added an issue https://github.com/rancher/dashboard/issues/11111